### DISCUSSION TITLE
"phantom ignore" permission request

### DISCUSSION THREAD

**Discussion by ArtifactyNight** [Zrips/CMI#2523] [Feature Requests]
**Description of feature/improvement request:**

put cmi.phantom.ignore to make phantom didn't spawn on player who have permission

### Detailed explanation of what it should do, when and in what situation: ###

phantom didn't spawning on player (have permission) even though the player didn't sleep for 3 days (in minecraft)

### Basic explanation as to why this feature is needed and/or useful to have: ###

make VIP rank to have this permission!

**Comment by Sikatsu** [comment]
In my case, it would be nice for staff to have. Everytime when we help players we bring a shit load of phantoms with us.

**Comment by Meldiron** [comment]
+1, would be really useful for me
